docs/architecture.md

```
@@ -86,10 +86,44 @@ All three audit-confirmed bugs are **fixed** by P0 bug-fix PR #471 (tracked by #
+  V458 --> G214["#214 lifecycle scoring + scheduled audited reads"]
+  R218["#218 Resident-only allocation + system-wide policy"] --> I215
+```
+
+**L2 contract freeze.** Before L3–L6 consumers land, freeze the versioned common event envelope/owner key; attempt identity and fingerprints; verifier verdict and `checkedPredicate`; stakes reference; Wait `ownerRef` and correlation; generic effect-intent/idempotency; schema/upcast/patch markers; and environment/nondeterminism manifests. Semantic changes require a new version/event and coordinated updates, never field re-meaning. First-consumer joins are fixed: #467's verifier registry → L3; #467 replay conformance consumes L6's fold/JSONL export before C1; #469 → L3; and #215 → L4.
+
+**C1 fixture — `C1-restart-refute-reconcile-replay`.** Resident creates WorkItem W with criteria and appends before delivery. Attempt A receives a unique `attemptId`/`attemptSeq`, sends an awaited granted message to existing agent E with no allocation delta, and appends a W-owned Wait. After the process exits, restart folds the Wait, correlates E's reply, and resumes without allocating a Worker. A submits known-bad evidence; #467 at L3 returns `refuted + checkedPredicate`, W remains incomplete, and no completion effect occurs. Retry B receives a distinct ID/sequence with `retryOf=A`; unchanged canonical task/model/environment let A and B share equivalence keys while both rows persist. B appends generic effect intent I, performs one idempotent fake effect, then crashes before confirmation; restart reconciles I under the same key, proves exactly one effect, and completes only on accepted evidence. JSONL/sidecars include attempt identity/fingerprints, Wait/reply, refutation, intent/reconcile, and completion. Replay by `replayKey` and nondeterminism manifest produces the identical commands and final fold, proves legacy upcast, fails loudly on a perturbed reducer or missing input, and performs zero live effects.
+
+The pre-existing C2 boundary remains #215 + #216; this roadmap synchronization does not expand its fixture.
+
+### Phase summary
 | Phase | Content |
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)